homeassistant/components/alarm_control_panel/manual.py

```diff
@@ -98,8 +98,10 @@ def state(self):
             elif (self._state_ts + self._pending_time +
                   self._trigger_time) < dt_util.utcnow():
                 if self._disarm_after_trigger:
+                    self._state = STATE_ALARM_DISARMED
```

Comment:
Fetching a property should never have any side effects like changing the value of `self._state`. Instead, the trigger should schedule a call to change the state back.